resources/rest-service/cloudify/migrations/versions/396303c07e35_5_2_to_5_3.py

```diff
@@ -23,9 +23,11 @@ def upgrade():
     _create_blueprints_labels_table()
     _modify_deployments_labels_table()
     _add_specialized_execution_fk()
+    _modify_filters_table()
 
 
 def downgrade():
+    _modify_filters_table()
```

Comment:
```suggestion
    _revert_changes_to_filters_table()
```

Reply:
I wonder how this one got there, thanks!